jujugui/static/gui/src/app/components/entity-header/entity-header.js

```diff
@@ -182,7 +175,43 @@ YUI.add('entity-header', function() {
           </div>
         );
       }
+      return deployAction;
+    },
 
+    /**
+      Generate the service, machine counts etc. for a bundle.
+    */
+    _generateCounts: function() {
+      var entity = this.props.entityModel.toEntity();
+      if (entity.type !== 'bundle') {
+        return;
+      }
+      var serviceCount = entity.serviceCount;
+      var unitCount = entity.unitCount;
+      var machineCount = entity.machineCount;
+      return (
+        <li>
+          {serviceCount} {this.props.pluralize('service', serviceCount)},&nbsp;
+          {machineCount} {this.props.pluralize('machine', machineCount)},&nbsp;
+          {unitCount} {this.props.pluralize('unit', unitCount)}
+        </li>);
```

Comment:
I had thought this too, but in #1514 Luca said it should look like: "By charmers • 5 charms, 5 machines, 5 units"